src/main/scala/org/hammerlab/guacamole/commands/VAFHistogram.scala

```diff
@@ -202,24 +202,27 @@ object VAFHistogram {
    * @param bins Number of bins to group the VAFs into
    * @return Map of rounded variant allele frequency to number of loci with that value
```

Comment:
Update the return docs? This is now map of `sampleName -> Map[Int, Long]` might be nice to have a type alias of `type SampleName = String`